cirq/ops/raw_types_test.py

```diff
@@ -111,3 +130,145 @@ def _decompose_(self, qubits):
 ))
 def test_gate_algebra(expression, expected_result):
     assert expression == expected_result
+
+
+def test_gate_shape():
+
+    class ShapeGate(cirq.Gate):
+
+        def _qid_shape_(self):
+            return (1, 2, 3, 4)
```

Comment:
Nit: unnecessary parens

Reply:
I think it's clearer with parentheses, especially for `return (2,)`.